apple/OmnivoreKit/Sources/App/Views/WebReader/WebReaderContainer.swift

```diff
@@ -68,6 +71,34 @@ struct WebReaderContainerView: View {
         .scaleEffect(navBarVisibilityRatio)
         Spacer()
       #endif
+      if FeatureFlag.enableTextToSpeechButton {
+        Button(
+          action: {
```

Comment:
Maybe we could move this action into audioSession? So we would just call something like `audioSession.handleUserTap` and then we could use it elsewhere.

Reply:
Yeah I tried to do that and then had to move it out, probably still could. There are two different play buttons displayed and they act a little differently. The play button on the top nav bar will play the current item, and if you are listening to the current item, pause it. But if you are listening to a different item and hit that play button, it will play the new item

The play button on the miniplayer just toggles play/pause though.